worker/httpserver/manifold.go

```diff
@@ -4,6 +4,7 @@
 package httpserver
 
 import (
+	stdcontext "context"
```

Comment:
Please be consistent with the naming, I've seen context, ctx, and stdcontext in different workers so far.